samza-rest/src/main/java/org/apache/samza/rest/proxy/job/SimpleYarnJobProxy.java

```diff
@@ -78,7 +79,7 @@ public void stop(JobInstance jobInstance)
     }
 
     String scriptPath = getScriptPath(jobInstance, STOP_SCRIPT_NAME);
-    int resultCode = scriptRunner.runScript(scriptPath, YarnCliJobStatusProvider.getQualifiedJobName(jobInstance));
+    int resultCode = scriptRunner.runScript(scriptPath, YarnRestJobStatusProvider.getQualifiedJobName(jobInstance));
```

Comment:
Just a minor suggestion. 

It looks odd to house getQualifiedName method in YarnRestJobStatusProvider class and invoke it just at scriptRunner.runScript as a param.
It could be better to either move that string generation to util method or move it to even jobInstance.toString(there by removing getQualifiedName method).